mscore/resourceManager.cpp

```diff
@@ -50,11 +50,13 @@ ResourceManager::ResourceManager(QWidget *parent) :
       languagesTable->setRowCount(1);
       languagesTable->setSpan(0, 0, 1, languagesTable->columnCount());
       languagesTable->setItem(0, 0, new QTableWidgetItem(tr("Loading…")));
+      worker = new PluginWorker(this);
       QtConcurrent::run(&workerThreads, worker, &PluginWorker::fetchLanguages);
       // display "loading" in pluginsTable
       pluginsTable->setRowCount(1);
       pluginsTable->setSpan(0, 0, 1, pluginsTable->columnCount());
       pluginsTable->setItem(0, 0, new QTableWidgetItem(tr("Loading…")));
+      worker = new PluginWorker(this);
```

Comment:
Does this free the new from a few lines abobe ot just loose the memory?

Reply:
`PluginWorker`s are freed using `deleteLater()` in places like:
https://github.com/musescore/MuseScore/blob/94a1703f944808232c92ac78e9d49ad41695fb86/mscore/plugin/pluginUpdater.cpp#L593
https://github.com/musescore/MuseScore/blob/94a1703f944808232c92ac78e9d49ad41695fb86/mscore/plugin/pluginUpdater.cpp#L606
https://github.com/musescore/MuseScore/blob/94a1703f944808232c92ac78e9d49ad41695fb86/mscore/plugin/pluginUpdater.cpp#L618
and
https://github.com/musescore/MuseScore/blob/94a1703f944808232c92ac78e9d49ad41695fb86/mscore/plugin/pluginUpdater.cpp#L645

Reply:
I added `deleteLater()` at the end of every function called using `QtConcurrent::run`.